rootfs/opt/fluentd/deis-output/lib/fluent/plugin/out_deis.rb

```diff
@@ -26,6 +26,10 @@ def initialize
       @influx_nsq = nil
       @log_topic = ENV['NSQ_LOG_TOPIC'] || "logs"
       @metric_topic = ENV['NSQ_METRIC_TOPIC'] || "metrics"
+      logs_to_nsq = ENV['SEND_LOGS_TO_NSQ'] || "true"
+      metrics_to_nsq = ENV['SEND_METRICS_TO_NSQ'] || "true"
+      @send_logs_to_nsq = if logs_to_nsq.downcase == 'false' then false else true end
```

Comment:
You're right let me think on this a bit. I'm not particularly thrilled with this - `@send_logs_to_nsq = if logs_to_nsq.downcase == 'false' then false else true end` its not easy to read